src/robotremoteserver.py

```diff
@@ -227,7 +238,11 @@ def get_keyword_documentation(self, name):
             return inspect.getdoc(self._library) or ''
         if name == '__init__' and inspect.ismodule(self._library):
             return ''
-        return inspect.getdoc(self._get_keyword(name)) or ''
+        keyword = self._get_keyword(name)
+        doc = inspect.getdoc(keyword) or ''
+        if len(getattr(keyword, 'robot_tags')):
```

Comment:
This causes `AttributeError` if keyword doesn't have `robot_tags`. I'll fix it myself along with few other minor changes and test enhancements.